components/EventCard/index.js

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+
+function EventCard({ event, teamName }) {
+  const options = { weekday: "short", month: "short", day: "2-digit" };
+  const eventDate = new Date(event.date).toLocaleDateString("de-DE", options).replace(/[.,]/g, "");
+  const eventTime = event.startTime + " - " + event.endTime;
+
+  return (
+    <Card>
+      <StyledDate data-testid="date" dateTime={event.date}>
+        {eventDate}
+      </StyledDate>
+      <p data-testid="time">{eventTime}</p>
+      <h3 data-testid="name">{event.name}</h3>
+      <p data-testid="team">{teamName}</p>
```

Comment:
See my comment above regarding test IDs.
Additionally, I think you could find more semantic elements than the `p` element as you are not really displaying paragraphs:

```suggestion
      <StyledDate dateTime={event.date}>
        {eventDate}
      </StyledDate>
      <time>{eventTime}</time>
      <h3>{event.name}</h3>
      <h4>{teamName}</h4>
```